models/contacts.js

```diff
@@ -1,19 +1,87 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const path = require("path");
 
-const listContacts = async () => {}
+const contactsPath = path.join(__dirname, "/contacts.json");
 
-const getContactById = async (contactId) => {}
+const listContacts = async () => {
+	try {
+		const data = await fs.readFile(contactsPath);
+		return data.toString();
+	} catch (err) {
+		console.log(err.message);
+	}
+};
 
-const removeContact = async (contactId) => {}
+const getContactById = async (contactId) => {
+	try {
+		const data = await fs.readFile(contactsPath);
+		const parsedData = JSON.parse(data);
+		const contact = parsedData.filter((el) => el.id === contactId);
+		return contact;
+	} catch (err) {
+		console.log(err.message);
+	}
+};
 
-const addContact = async (body) => {}
+const removeContact = async (contactId) => {
+	const contactList = await fs.readFile(contactsPath);
+	const parsedContactList = JSON.parse(contactList);
+	const contactIndex = parsedContactList.findIndex(
```

Comment:
you already know `contactId`. Its in method parameter. Why not getContects, then filter them excluding the one with provided ID and store in json file again? Sounds simpler in my opinion.